src/main/kotlin/app/revanced/patches/reddit/customclients/boostforreddit/api/patch/ChangeOAuthClientIdPatch.kt

```diff
@@ -1,35 +1,31 @@
 package app.revanced.patches.reddit.customclients.boostforreddit.api.patch
 
-import app.revanced.patcher.annotation.*
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
 import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
 import app.revanced.patcher.fingerprint.method.impl.MethodFingerprintResult
 import app.revanced.patcher.patch.PatchResult
 import app.revanced.patcher.patch.PatchResultSuccess
 import app.revanced.patcher.patch.annotations.Patch
 import app.revanced.patches.reddit.customclients.AbstractChangeOAuthClientIdPatch
+import app.revanced.patches.reddit.customclients.ChangeOAuthClientIdPatchAnnotation
 import app.revanced.patches.reddit.customclients.boostforreddit.api.fingerprints.GetClientIdFingerprint
 
 @Patch
```

Comment:
I think moving @Patch out of it shouldn't be necessary. Instead this issue should be fixed in ReVanced Patcher.

Reply:
Yeah I guess that would be a better fix